components/Header/Header.tsx

```diff
@@ -135,27 +135,25 @@ export const Header: React.FC<HeaderProps> = () => {
     <>
       <MantineHeader height={60} className={classes.root}>
         <Container className={classes.container} fluid>
-          <Group spacing="xs">
-            <Image
-              src="/logo/android-icon-36x36.png"
-              width={30}
-              height={30}
-              alt={"COH3 Stats logo"}
-            />
-            <Title order={1} size="h3">
-              COH3 Stats
-            </Title>
-          </Group>
+          <Link href={"/"}>
+            <Group spacing="xs">
+              <Image
+                src="/logo/android-icon-36x36.png"
+                width={30}
+                height={30}
+                alt={"COH3 Stats logo"}
+              />
+
+              <Title order={1} size="h3">
+                COH3 Stats
```

Comment:
![image](https://user-images.githubusercontent.com/8086995/221427997-a3942a3f-cc35-4ec9-ad53-1dccb2a017a4.png)
I don't know how to fix this CSS, @JohannesMerkt